Jenkinsfile.release

```diff
@@ -172,5 +175,33 @@ podTemplate(cloud: 'openshift', label: pod_label, yaml: pod) {
                 }
             }
         }
+
+        if (ostree_prod_refs.size() > 0) {
+            stage("OSTree Import: Wait and Verify") {
+                def tmpd = shwrapCapture("mktemp -d")
+
+                shwrap("""
+                cd ${tmpd}
+                ostree init --mode=archive --repo=.
+                cat /etc/ostree/remotes.d/fedora.conf >> config
+                """)
+
+                timeout(time: 20, unit: 'MINUTES') {
```

Comment:
let's add a comment here on why we need the timeout. basically it takes some time for the infra changes to propagate and the summary file to get updated